### DISCUSSION TITLE
Developer Tools: make service data textarea as wide as the page itself

### DISCUSSION THREAD

**Discussion by sreusr** [home-assistant/frontend#7386] [Accessibility] [answered]
There's enough empty space to the right, why not having a wider textarea so long lines are readable and don't need to line break.

**Comment by apaperclip** [comment]
I can make a separate issue/discussion if needed but I would like to add: all of the dev tool pages could use a spacing review. E.g. The states table column is too narrow for entity_id. Most of my entity_ids have breaks.

**Reply by spacegaier** [reply]
I am currently optimizing the space a bit for the dev tools.

However, I will not change the line-breaking of the entity IDs because forcing it to be in a single line, would be bad I think. Since each entity has multiple attributes we anyway need multiple lines. So why not make use of that space?

**Comment by spacegaier** [comment, marked as answer]
The "Services" dev tool tab has been completely reworked some time ago, so this discussed issue no longer exists: https://github.com/home-assistant/frontend/pull/8410
